post_installation_configuration/aws-compute-edge-tasks.adoc

```diff
@@ -0,0 +1,98 @@
+:_content-type: ASSEMBLY
+:context: aws-compute-edge-tasks
+[id="aws-compute-edge-tasks"]
+= AWS Local Zone tasks
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+After installing {product-title} in AWS, you can further expand and customize your cluster
+to your requirements with support of Local Zones and edge compute pool.
+
+[id="extend_existing_to_local_zones_day2"]
+== Extend existing clusters to use AWS Local Zones
+
+You can extend an existing {product-title} installation in AWS to use AWS Local Zones for Day 2 operations.
+
+To extend nodes to Local Zone locations, you must create a subnet in the VPC for the target zone,
+adjust the cluster network MTU, and create the Machine set manifest to create the node
+into that location.
+
+[role="_additional-resources"]
+.Additional resources
+
+* See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation for more information about AWS Local Zones and the supported instances types and services.
+
+.Prerequisites
+
+* You must check the free CIDR Blocks available on the VPC
+
+// Edge Compute Pool
+include::modules/machines-edge-machine-pool.adoc[leveloffset=+2]
+
+// Opt-In Zone
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+
+[id="extend_existing_to_local_zones_day2_mtu"]
+=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+
+.Prerequisites
```

Comment:
This prereq heading won't pass modular documentation standards. A lead in sentence will be needed that introduces this section's purpose to a doc user. The prereq could be converted to an IMPORTANT notice.